java/display-kml/README.metadata.json

```diff
@@ -12,6 +12,12 @@
         "Portal",
         "PortalItem"
     ],
+    "provision_from": [
+        "https://arcgisruntime.maps.arcgis.com/home/item.html?id=324e4742820e46cfbe5029ff2c32cb1f"
+    ],
+    "provision_to": [
+        "/US_State_Capitals.kml"
```

Comment:
looking at the path for this data in the sample, I think this should be `/ArcGIS/Samples/KML/US_State_Capitals.kml`?